src/api/customization.rs

```diff
@@ -42,6 +44,42 @@ pub trait Customization {
     /// This can improve privacy, but can make usage less comfortable.
     fn default_cred_protect(&self) -> Option<CredentialProtectionPolicy>;
 
+    /// Sets the initial minimum PIN length in code points.
+    ///
+    /// # Invariant
+    ///
+    /// - The minimum PIN length must be at least 4.
+    /// - The minimum PIN length must be at most 63.
+    /// - default_min_pin_length_rp_ids() must be non-empty if MAX_RP_IDS_LENGTH is 0.
```

Comment:
nit: `MAX_RP_IDS_LENGTH` is now `max_rp_ids_length` but we could as well do a pass on documentation after the code change if it's easier (there's a few other cases like this one below, I'm not commenting on them to minimize comments to those regarding design).

Reply:
done. @ia0 are there more that needs to be improved in this CL or can this be approved and merged?

Reply:
I wanted to test a bit the optimizations to see if there was a way to make it work with `Vec<String>` too. But actually ever `Vec<&'static str>` doesn't seem to work. Probably the `iter()` is too much.